shared/common-adapters/usernames/index.js

```diff
@@ -19,6 +19,7 @@ export type BaseUsernamesProps = {|
   backgroundMode?: Background,
```

Comment:
I show up as orange with the flag off, intended?

Reply:
fixed, actually colorYou was being set to orange always in name with icon